packages/shadycss/package.json

```diff
@@ -2,7 +2,8 @@
   "name": "@webcomponents/shadycss",
   "version": "1.9.6",
   "description": "Polyfill for Scoped CSS",
-  "main": "shadycss.min.js",
+  "main": "interface.js",
+  "module": "interface.js",
```

Comment:
I think this would be a breaking change to this package.

Reply:
Good catch!

Reply:
I don't think so, because there is no `shadycss.min.js`, so it must have been a broken import for a long time: https://unpkg.com/browse/@webcomponents/shadycss/

That said, WDYT about this module being the default import vs say `@webcomponents/shadycss/interface.js`? I could imagine expecting `@webcomponents/shadycss` would import the polyfill (even though right now that does nothing).

Reply:
I like this module as the main module - it discourages accidentally loading the polyfill.

I do think it'd be interesting to see if the interface module could load the polyfill if it's needed, or at least warn if it's needed and not loaded. That could help with components used in contexts where the app doesn't necessarily know they're using shadow DOM.